.github/workflows/generated-macos-arm64-binary-wheel-nightly.yml

```diff
@@ -139,7 +139,7 @@ jobs:
       #       favor of GPU_ARCH_VERSION
       DESIRED_CUDA: cpu
       GPU_ARCH_TYPE: cpu
-      DOCKER_IMAGE: pytorch/manylinux2_28-builder:cpu-main
```

Comment:
I believe docker images are not used for MacOS builds. TODO: Refactor code to remove this parameter since its confusing.